README.md

```diff
@@ -1,2 +1,321 @@
+
+Options:
+  --help  Show this message and exit.
+  
+Example:
+  qne application validate
+```
+</details>
+
+
+
+<!--- QNE EXPERIMENT CREATE --->
+<details closed>
+<summary><b>qne experiment create</b></summary>
+Create a new experiment, based on either a local or a remote application name.
+<br></br>
+    
+```
+qne experiment create [OPTIONS] NAME APPLICATION NETWORK
+
+Arguments:
+  NAME         Name of the experiment.  [required]
+  APPLICATION  Name of the application.  [required]
+  NETWORK      Name of the network to be used. [required]
+
+Options:
+  --local / --remote  Run the application locally.  [default: True]
+  --help              Show this message and exit.
+  
+Example:
+  qne experiment create experiment_name application_name europe
+```
+</details>
+
+
+
+<!--- QNE EXPERIMENT VALIDATE --->
+<details closed>
+<summary><b>qne experiment validate</b></summary>
+Validates whether the experiment file structure is  complete and if the json content is valid.
+<br></br>
+
+```
+qne experiment validate [OPTIONS]
+
+Options:
+  --help  Show this message and exit.
+  
+Example:
+  qne experiment validate
+```
+</details>
+
+
+<!--- QNE EXPERIMENT DELETE --->
+<details closed>
+<summary><b>qne experiment delete</b></summary>
+Delete the entire experiment, both on the local and remote side.
+<br></br>
+    
+```
+qne experiment delete [OPTIONS]
+
+Options:
+  --help  Show this message and exit.
+  
+Example:
+  qne experiment delete
+```
+</details>
+
+
+
+<!--- QNE EXPERIMENT LIST --->
+<!---
+<details closed>
+<summary><b>qne experiment list</b></summary>
+List all the remote applications.
+<br></br>
+    
+```
+qne experiment list [OPTIONS]
+
+Options:
+  --help  Show this message and exit.
+  
+Example:
+  qne experiment list
+```
+</details>
+--->
+
+
+
+<details closed>
+<!--- QNE EXPERIMENT RUN --->
+<summary><b>qne experiment run</b></summary>
+Using this command the experiment will be run on the backend. In case of a local run, netsquid will be used as backend simulator. 
+<br></br>
+
+```
+qne experiment run [OPTIONS]
+
+Options:
+  --block  Wait for the result to be returned.  [default: False]
+  --help   Show this message and exit.
+  
+Example:
+  qne experiment run
+```
+</details>
+
+
+
+<!--- QNE EXPERIMENT RESULTS --->
+<details closed>
+<summary><b>qne experiment results</b></summary>
+Download the results for an experiment that has been run.
+<br></br>
+    
+```
+qne experiment results [OPTIONS]
+
+Options:
+  --all   Get all results for this experiment.  [default: False]
+  --show  Show the results on screen instead of saving to file.  [default:
+          False]
+  --help  Show this message and exit.
+  
+Example:
+  qne experiment results
+```
+</details>
+
+
+
+## More documentation
+More documentation about these commands and about the files that are generated can be found in the QNE-ADK user guide on the Quantum Network Explorer [website](https://beta.quantum-network.com/knowledge-base/qne-adk).
```

Comment:
1. [website] -> [knowledge base]
2. This link is not correct. beta.quantum...??